### DISCUSSION TITLE
More themes that behave like "dark:" class

### DISCUSSION THREAD

**Discussion by kagiorgis** [tailwindlabs/tailwindcss#9564] [Help]
So, I want to create a third theme called **retro** (light, dark, retro).

I don't want to do that by defining new colours, I just want that theme to behave exactly like the dark theme, e.g. 
`className="text-black dark:text-white retro:text-brown"
`

Any ideas on how to do that?

**Comment by wongjn** [comment]
By default, the `dark:` uses the `prefers-color-scheme` media query but there is not such thing for any user/author-defined theme name. Instead you would have to rely on something like adding a `retro` class to your `<html>` element when this theme should be activated. In doing so, you could then write a [custom plugin to add a new variant](https://tailwindcss.com/docs/plugins#adding-variants) called `retro` that matches when a parent element has this class:
```js
const plugin = require('tailwindcss/plugin');

module.exports = {
  // …
  plugins: [
    plugin(({ addVariant }) => {
      addVariant('&:is(.retro *)');
    }),
  ],
};
```

**Reply by kagiorgis** [reply]
Thanks a lot for your reply!

All I want basically is to use the "hidden" and "block" rules to hide or unhide an entire div when retro is selected. How would I do that and would your solution work?

Thanks in advance :)

**Reply by wongjn** [reply]
You may not need a custom variant at all in that case – you could add or remove the `hidden` class to the necessary `<div>` elements when retro is selected. My solution would work too.
